src/controllers/roomSocket.ts

```
@@ -52,13 +44,15 @@ function joinRoom(socket: Socket, data: { code: string, joinAs: "player" | "god"
                 )
             }
 
-            await clientDB.collection('rooms').findOne({code: data.code}).then((updatedRoom) => {
-                socket.join(data.code);
-                socket.emit('rooms:join', updatedRoom);
+            await clientDB.collection('rooms').findOne({code: data.code}).then(async (updatedRoom) => {
+                if (updatedRoom && updatedRoom.creator) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)